extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -142,6 +143,49 @@ function () {
 		return $data;
 	}
 
+
+	/**
+	 * Get $limit chat moderators who have contributed most recently
+	 * filter out bots and admins
+	 *
+	 * @param $limit
+	 * @return array
+	 */
+	public function getChatModerators( $limit ) {
+		$data = WikiaDataAccess::cache(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$data". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)